sdk/jobs/automl-standalone-jobs/automl-image-classification-multiclass-task-fridge-items/automl-image-classification-multiclass-task-fridge-items.ipynb

```diff
@@ -427,20 +505,6 @@
     "If you wish to use the default hyperparameter values for a given algorithm (say `vitb16r224`), you can specify the job for your AutoML Image runs as follows:"
    ]
   },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {
-    "tags": [
-     "parameters"
-    ]
-   },
-   "outputs": [],
-   "source": [
-    "# set up experiment name\n",
-    "exp_name = \"dpv2-image-classification-experiment\""
-   ]
-  },
```

Comment:
Not sure if we want to define the `image_classification_job` for each of the sections... seems repetitive... just a thought, feel free to keep it as it is in case you feel otherwise...

Reply:
It is repetitive, but the audience is users unfamiliar with AutoMode and I'm guessing repetition is good for the learning process. Since repetition already exists independently of AutoMode, added this point to https://msdata.visualstudio.com/DefaultCollection/Vienna/_workitems/edit/1977845 .